src/NodaTime/Duration.cs

```diff
@@ -61,6 +61,7 @@ namespace NodaTime
     /// </remarks>
     /// <threadsafety>This type is an immutable value type. See the thread safety section of the user guide for more information.</threadsafety>
     [TypeConverter(typeof(DurationTypeConverter))]
+    [XmlSchemaProvider(nameof(GetSchema))]
```

Comment:
Given that we get to pick the name here, could we use `AddSchema`? It feels like that describes what it's doing more than `GetSchema`.

Reply:
Actually, the method does both. It adds the schema to the provided `XmlSchemaSet` and returns its qualified name. The documentation about the [XmlSchemaProviderAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.xml.serialization.xmlschemaproviderattribute) uses `MySchema` in its example which is even worse than `GetSchema` and `AddSchema`.
But given that all the internal methods are named `Add<NodaType>SchemaType` probably `AddSchema` makes more sense. I'll change the names to `AddSchema`.